docs/dev-guide/integration/rest-api/repo/cud.rst

```diff
@@ -155,6 +155,12 @@ dictate how the importer will function when it synchronizes content. The possibl
 configuration values are contingent on the type of importer being added; each
 importer type will support a different set of values relevant to how it functions.
 
+.. note::
+        There is an optional importer configuration parameter that is usable
+        on all importer plugin types called ``force_full``. If this parameter is set,
+        every sync of the repo using this importer will be done from scratch.
+        More details are `here <http://pulp.readthedocs.org/en/latest/user-guide/release-notes/2.9.x.html#pulp-2-9-0>`_.
```

Comment:
I am not sure I understand what is sync "from scratch", you probably meant that sync will be operational. Is my understanding correct?